history/cultures/titanforged_group.txt

```diff
@@ -17,6 +21,7 @@
 	#
 	discover_innovation = innovation_taming_the_skies
 	discover_innovation = innovation_large_soldiers
+	discover_innovation = innovation_giants
```

Comment:
They need their own unit type so bad.
Basically, there shouldn't be generic giant and flying beast MaAs, only special to cultures.
You may do it on another branch but write it in your to-do list with a priority.

Reply:
Would Iron Giants be a good fit? Also can I use Keepers as a Titanforged Knight title?